src/styles/main.scss

```
@@ -1,3 +1,204 @@
-body {
-  margin: 0;
+.page {
+  font-family: "Roboto", sans-serif;
+
+  &__body {
+    margin: 0;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.img-logo {
+
+  &__main-page {
+    display: block;
+    font-size: 0;
+  }
+}
+
+.navigation {
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+
+    &-item {
+      list-style: none;
+
+      &:first-child a {
+        margin-left: 0;
+      }
```

```suggestion
      &-link:first-child {
        margin-left: 0;
      }
```
Замість використання такого міксу, можна просто використати той клас, якому ти задаєш margin-left